libs/constants/poolLists.ts

```diff
@@ -0,0 +1,235 @@
+                decimals: 6,
+            },
+        },
+        {
+            name: '3-BTC/USD',
+            address: '0x70988060e1FD9bbD795CA097A09eA1539896Ff5D',
+            leverage: 3,
+            updateInterval: ONE_HOUR,
+            frontRunningInterval: FIVE_MINUTES,
+            keeper: '0x759E817F0C40B11C775d1071d466B5ff5c6ce28e',
+            longToken: {
+                name: '3-LONG-BTC/USD',
+                address: '0x05A131B3Cd23Be0b4F7B274B3d237E73650e543d',
+                symbol: '3L-BTC/USD',
+                decimals: 6,
+            },
+            shortToken: {
+                name: '3-SHORT-BTC/USD',
+                address: '0x85700dC0bfD128DD0e7B9eD38496A60baC698fc8',
+                symbol: '3S-BTC/USD',
+                decimals: 6,
+            },
+            quoteToken: {
+                name: 'USDC',
+                symbol: 'USDC',
+                address: '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8',
+                decimals: 6,
+            },
+        },
+        {
+            name: '1-ETH/USD',
+            address: '0x3A52aD74006D927e3471746D4EAC73c9366974Ee',
+            leverage: 1,
+            updateInterval: ONE_HOUR,
+            frontRunningInterval: FIVE_MINUTES,
+            keeper: '0x759E817F0C40B11C775d1071d466B5ff5c6ce28e',
+            longToken: {
+                name: '1-LONG-ETH/USD',
+                address: '0x38c0a5443c7427e65A9Bf15AE746a28BB9a052cc',
+                symbol: '1L-ETH/USD',
+                decimals: 6,
+            },
+            shortToken: {
+                name: '1-SHORT-ETH/USD',
+                address: '0xf581571DBcCeD3A59AaaCbf90448E7B3E1704dcD',
+                symbol: '1S-ETH/USD',
+                decimals: 6,
+            },
+            quoteToken: {
+                name: 'USDC',
+                symbol: 'USDC',
+                address: '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8',
+                decimals: 6,
+            },
+        },
+        {
+            name: '3-ETH/USD',
+            address: '0x54114e9e1eEf979070091186D7102805819e916B',
+            leverage: 3,
+            updateInterval: ONE_HOUR,
+            frontRunningInterval: FIVE_MINUTES,
+            keeper: '0x759E817F0C40B11C775d1071d466B5ff5c6ce28e',
+            longToken: {
+                name: '3-LONG-ETH/USD',
+                address: '0xaA846004Dc01b532B63FEaa0b7A0cB0990f19ED9',
+                symbol: '3-ETH/USD',
+                decimals: 6,
+            },
+            shortToken: {
+                name: '3-SHORT-ETH/USD',
+                address: '0x7d7E4f49a29dDA8b1eCDcf8a8bc85EdcB234E997',
+                symbol: '3S-ETH/USD',
+                decimals: 6,
+            },
+            quoteToken: {
+                name: 'USDC',
+                symbol: 'USDC',
+                address: '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8',
+                decimals: 6,
+            },
+        },
+    ],
+    [ARBITRUM_RINKEBY]: [
```

Comment:
I couldn't find a source of truth to cross check the testnet config against, do you know if its in notion somewhere or did you just query it on chain to the get the deets?

Reply:
is fast updateInterval https://www.notion.so/tracerdao/Pools-Deployments-fbf7f297f16d49f2ac02d7cb9700a920#f10f816ad57d40d7a14d81cdbb6131c6